apis/validator/duties/sync.yaml

```diff
@@ -0,0 +1,52 @@
+post:
+  tags:
+    - ValidatorRequiredApi
+    - Validator
+  summary: "Get sync committee duties"
+  operationId: "getSyncCommitteeDuties"
+  description: "Requests the beacon node to provide a set of sync committee duties for a particular epoch."
+  parameters:
+    - name: epoch
+      description: "Should only be allowed 1 epoch ahead"
```

Comment:
I guess this is leftover from copying the attester duties API spec.

I think we should definitely specify a bound on the `epoch`, but I think the correct one requires `EPOCHS_PER_SYNC_COMMITTEE_PERIOD` to be expressed properly:

```python
epoch // EPOCHS_PER_SYNC_COMMITTEE_PERIOD <= current_epoch // EPOCHS_PER_SYNC_COMMITTEE_PERIOD + 1
```

i.e. `epoch` must be within the next sync committee period, or earlier

Reply:
I had removed the description implementing, but hadn't updated here because I wasn't sure how to express it formally... I can copy/paste this description and go from there...